src/main/webapp/assets/feedScript.js

```diff
@@ -77,15 +77,24 @@ document.addEventListener('DOMContentLoaded', onLoad);
 /**
  * Fires as soon as the DOM is loaded.
  */
-function onLoad() {
+async function onLoad() {
+  const collegeid = (new URLSearchParams(window.location.search)).get('collegeid');
+
+  // Don't load the map, posts if there wasn't a college ID provided.
+  if (!collegeid) {
+    document.getElementById('find-events-title').innerHTML =
+      'No college provided. Please visit the <a href="/">home page</a> to pick a college.';
+    return;
+  }
+
   // In the future, there will be real GET requests here, but for now, just fake ones.
   // These global variables will be assigned here and never assigned again.
-  posts = fetchFakePosts();
-  collegeLocation = fetchFakeCollegeLocation();
+  posts = fetchFakePosts(collegeid);
+  collegeLocation = await fetchFakeCollegeLocation(collegeid);
 
-  if (document.getElementById('all-posts')) {
-    addPosts(posts);
-  }
+  // Update text elements on page with fetched information.
+  document.getElementById('find-events-title').innerText += ` @ ${collegeLocation.name}`;
```

Comment:
We could also convert the name to lowercase to match with the ~aesthetic~

Reply:
okay, sounds good!